### DISCUSSION TITLE
Ignoring "Binary files are different"

### DISCUSSION THREAD

**Discussion by JGP999** [WinMerge/winmerge#2411] [General]
I used robocopy to copy over thousands of files from an ext HDD to a sharepoint library (which I mapped as a network drive). Once finished I looked at the properties of both folders and saw that there were more files in the source so I used WinMerge to tell me what was missing.
There are hundreds and hundreds of comparison results telling me the Binary files are different. When I check random selections from the destination they open and display fine. No one has edited these files yet.
I'm not sure why these comparisons say this and it is not all as equally hundreds say "Binary files are identical"

1. Any idea why this would happen?
2. Any way to get WinMerge just to output which files do not exist in the destination and to ignore and binary differences?

Many thanks

**Comment by JGP999** [comment]
I think I may have solved this by unchecking "show different items" under the view menu... at least this removes "binary files are different" but not "binary files are identical" which is a good start.
Update: And now If I uncheck show identical items I am left with files that exist in A but not B (or vice versa) which is exactly what I need.
Hooray!

**Comment by JGP999** [comment]
Found my solution as per follow up comment

**Reply by sdottaka** [reply]
> Any idea why this would happen?

When I opened an old format Office file, the contents of the file would change even though I hadn't made any changes. Maybe something similar to this happened.

> Any way to get WinMerge just to output which files do not exist in the destination and to ignore and binary differences?

I'm glad this has already been resolved.
